### ISSUE TITLE
🚀 Feature: Hide Support button if it's not configured

### ISSUE THREAD

**Issue by drodil** [backstage/backstage#15821]
### 🔖 Feature description

When support is not configured in the app config, the support button should be hidden at least in production

### 🎤 Context

Having support button without any user interaction possibility as it is in the hands of the app owner is not good UX.

### 👀 Have you spent some time to check if this feature request has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

**Comment by Rugvip** [comment]
Makes sense to me, it's not terribly useful at this point without config. What do you think @backstage/techdocs-core?

**Comment by emmaindal** [comment]
Totally agree, is this something you would like to work on @drodil?

**Comment by efenner-cambia** [comment]
I'm a fan of this change for usability, but without the empty search button in the UI to prompt implementation, is there sufficient documentation to nudge and direct adopters toward adding it back?

**Comment by emmaindal** [comment]
@efenner-cambia great call out - maybe its something we can add to the documentation of `App configuration` in the [getting started guide](https://backstage.io/docs/getting-started/configure-app-with-plugins) . What do you think?

**Comment by Rugvip** [comment]
@efenner-cambia yep very good point. There's definitely part of me that thinks we can't really do this yet, i.e. there are some patterns and tools that we need to find in the frontend system before we're able to do this kind of configuration in a homogenous way.

**Comment by efenner-cambia** [comment]
I think there are multiple places in the documentation where we could better highlight "here's an example of a best practice that you can modify as needed". In this case it has been self documenting because you see it in the UI, but if we take that away then I just think it's worth making sure that the documentation will point users/adopters in the right direction.

**Comment by moley** [comment]
Hi to all, are there any plans on this issue to be fixed?

**Comment by benjdlambert** [comment]
@moley we would like to fix it, but it's not part of our backlog right now - We're happy to review contributions though in this area if someone would like to pick this up :pray:

**Comment by nebulon42** [comment]
This seems to have been solved in #27144 but the issue was not closed (yet).
